tests/integ/test_simplifier_suite.py

```diff
@@ -1618,17 +1624,17 @@ def test_select_limit_orderby(session):
     df1 = df.select("a", "b").limit(2).sort(col("a"))
     Utils.check_answer(df1, [Row(3, "b"), Row(5, "a")])
     # sql simplification is not applied, and order by clause is attached at end
-    expected_query = """SELECT  *  FROM ( SELECT "A", "B" FROM ( SELECT $1 AS "A", $2 AS "B" FROM  VALUES (5 :: INT, 'a' :: STRING), (3 :: INT, 'b' :: STRING)) LIMIT 2) ORDER BY "A" ASC NULLS FIRST"""
+    expected_query = """SELECT \n     * \n FROM (\n SELECT \n    "A", \n    "B"\n FROM (\n SELECT $1 AS "A", $2 AS "B" FROM  VALUES (5 :: INT, 'a' :: STRING), (3 :: INT, 'b' :: STRING)\n)\n LIMIT 2\n)\n ORDER BY \n    "A" ASC NULLS FIRST"""
```

Comment:
How about introducing a new function that compares two strings by replacing all whitespaces with a single space. That way we don't have to add `\n` and tabs in the test files. instead we can just do `compare_normalized_strings(df.queries[0], expected)`